crates/biome_migrate/tests/specs/migrations/nurseryRules/middleToExistingGroup.json.snap

```diff
@@ -0,0 +1,52 @@
+---
+source: crates/biome_migrate/tests/spec_tests.rs
+expression: middleToExistingGroup.json
+---
+# Input
+```json
+{
+  "linter": {
+    "rules": {
+      "nursery": {
+        "nuseryRuleAlways": "error",
+        "noExcessiveNestedTestSuites": "error",
+        "nuseryRuleForever": "error"
+      },
+      "complexity": {}
+    }
+  }
+}
+
+```
+
+# Diagnostics
+```
+middleToExistingGroup.json:5:36 migrate  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! This rule has been promoted to complexity/noExcessiveNestedTestSuites.
+  
+    3 │     "rules": {
+    4 │       "nursery": {
+  > 5 │         "nuseryRuleAlways": "error",
+      │                                    ^
+  > 6 │         "noExcessiveNestedTestSuites": "error",
+      │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Also a question, should I include the comma into the text range? This doesn't look very nice.

Reply:
I excluded comma from the text range.